monitoring/README.md

```diff
@@ -70,36 +70,34 @@ command `git clone <https_url>`.
-to be stored outside of the directory structure of this repository. The default
-`USER_DIR` is the root of this repository. A directory referenced by `USER_DIR`
-should include `user*` files in the same relative structure as they exist in
-this repository.
+See the [main README](../README.md) to for information about the customization process.
 
-The following files are automatically used by the monitoring scripts if they
-are present in `USER_DIR`:
+After you create the location for your customization files, you can customize the components that you deploy for monitoring by specifying environment variables and Helm chart parameters in this set of customization files (in this example, contained in the *my-viya4mon-user-dir* root directory):
 
-* `user.env`
-* `monitoring/user.env`
-* `monitoring/user-values-prom-operator.yaml`
-* `monitoring/user-values-pushgateway.yaml`
+```text
+/my-viya4mon-user-dir/user.env
 
-You can modify two types of values to customize your deployment:
-- Environment variables: These variables can be specified in the `user.env` file or on the command line. Specifying the variables in `user.env` is recommended, in order to maintain a consistent set of values for future deployments.
-- Helm chart parameters: These parameters are organized into a hierarchical structure (for example, `persistentVolume:storageClass`). These parameters are specified in `.yaml` configuration files.
+/my-viya4mon-user-dir/monitoring/user.env
+/my-viya4mon-user-dir/monitoring/user-values-prom-operator.yaml
+/my-viya4mon-user-dir/monitoring/user-values-pushgateway.yaml
+```
+
+You specify the environment variables in the `user.env` files and the Helm chart parameters in the `*.yaml` configuration files. 
+
+#### Specifying Environment Variables in user.env Files
 
-### user.env
+Environment variables control script behavior and high-level options such as TLS and workload node placement. Note that you can also specify environment variables on a command line, but specifying the variables in `user.env` is recommended, in order to maintain a consistent set of values for future deployments. The values in the top-level `user.env` file (`/my-viya4mon-user-dir/user.env`) apply to both the monitoring and logging deployments. The values in `/my-viya4mon-user-dir/monitoring/user.env` apply only to the monitoring deployment.
 
-The `monitoring/user.env` file contains environment variables that customize
-the components that are deployed or alter some script behavior (such as to
-enable debug output). All values in `user.env` files are exported as environment
-variables available to the scripts. Any line whose first character is `#` is treated as a comment and ignored.
+Any line whose first character is `#` is treated as a comment and ignored.
 
 You can set the `GRAFANA_ADMIN_PASSWORD` environment variable to specify the default password for Grafana. If you do not specify a default password, one is randomly generated and displayed during deployment.
```

Comment:
I wonder if we should highlight this in some way...maybe with a sub-header (Changing the Administrator Password) or enclosing it in a box or something.  Seems like an important thing to call people's attention to even if they don't think they want to do any customizations.

Reply:
done